com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/HDRenderPipeline.VolumetricCloudsSky.cs

```diff
@@ -0,0 +1,360 @@
+using UnityEngine.Experimental.Rendering;
+using UnityEngine.Experimental.Rendering.RenderGraphModule;
+
+namespace UnityEngine.Rendering.HighDefinition
+{
+    public partial class HDRenderPipeline
+    {
+        // Intermediate textures required for baking the clouds into the cubemap
+        RTHandle m_IntermediateCloudsLightingBuffer = null;
+        RTHandle m_IntermediateCloudsDepthBuffer = null;
+        RTHandle m_IntermediateCloudsLightingCube0Buffer = null;
+        RTHandle m_IntermediateCloudsLightingCube1Buffer = null;
+        MaterialPropertyBlock m_MpbClouds = new MaterialPropertyBlock();
+
+        void ReleaseVolumetricCloudsStaticTextures()
+        {
+            if (m_Asset.currentPlatformRenderPipelineSettings.supportVolumetricClouds)
+            {
+                RTHandles.Release(m_IntermediateCloudsLightingBuffer);
+                RTHandles.Release(m_IntermediateCloudsDepthBuffer);
+                RTHandles.Release(m_IntermediateCloudsLightingCube0Buffer);
+                RTHandles.Release(m_IntermediateCloudsLightingCube1Buffer);
+            }
+        }
+
+        void InitializeVolumetricCloudsStaticTextures()
+        {
+            if (m_Asset.currentPlatformRenderPipelineSettings.supportVolumetricClouds)
+            {
+                int skyResolution = (int)m_Asset.currentPlatformRenderPipelineSettings.lightLoopSettings.skyReflectionSize;
```

Comment:
This buffers are only used to render the sky in order to compute ambient probe for the clouds right?
If that's the case, I think we should not use skyReflectionSize here because it can be huge (2k or 4k) and it would be very wasteful, even when this goes through render graph.
There are plans to add a new resolution setting for ambient only when possible but in the meantime, can we use a much smaller hardcoded fixed size here?